recipe/meta.yaml

```diff
@@ -1,60 +1,76 @@
 {% set name = "papermill" %}
-{% set version = "0.14.2" %}
-{% set bundle = "tar.gz" %}
-{% set hash_type = "sha256" %}
-{% set hash = "60a282ffdcc14ae427a85a12fbd856885716010ef691066f2f1957e5904be52d" %}
-
+{% set version = "0.19.0" %}
 package:
-  name: {{ name|lower }}
-  version: {{ version }}
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
 
 source:
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ bundle }}
-  {{ hash_type }}: {{ hash }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d62d7edcb635040a84ccf42b6dcc7e9163a8945059c8eee5211218f56d8f3904
 
 build:
-  number: 1000
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  number: 0
   entry_points:
     - papermill = papermill.cli:papermill
-  
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
 requirements:
   host:
+    - pip
     - python
-    - pip >=9.0.1
-
   run:
-    - python
     - ansiwrap
-    - boto3
+    - backports.tempfile
     - click
+    - entrypoints
     - future
-    - futures  # [py2k]
+    - futures
     - ipython >=5.0
-    - pyyaml
-    - nbformat
-    - nbconvert
-    - six
-    - tqdm
     - jupyter_client
+    - nbconvert >=5.4
+    - nbformat
     - pandas
+    - python
+    - pyyaml
     - requests
+    - six
+    - tqdm >=4.29.1
+    - pip
 
 test:
+  requires:
```

Comment:
Looks like `retry` was missed here.

Reply:
Also wish these could read requirement.txt file from the repo so it doesn't have to repeat itself :/

Reply:
Retry was erroring so I pulled it out. I'm going to ping some conda-forge friends on Monday if I have time.

Reply:
@willingc I made a PR (linked below) to remove retry dependency from papermill. I can do a patch release to unblock this PR hopefully after it merges.

Reply:
That would be awesome! Thanks @MSeal

Reply:
@willingc If you want to bump to 0.19.1, the failure on retry not existing should go away.